From foo@baz Mon Aug 17 17:22:08 PDT 2009
Date: Mon, 17 Aug 2009 17:22:08 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: hv: remove Sources.c

It's a .c file including other .c files, ick.
Remove that mess now that the header files are unwound.

Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/Channel.c          |    1 +
 drivers/staging/hv/ChannelInterface.c |    4 ++++
 drivers/staging/hv/ChannelMgmt.c      |    2 ++
 drivers/staging/hv/Connection.c       |    4 +++-
 drivers/staging/hv/Hv.c               |    3 ++-
 drivers/staging/hv/Makefile           |    4 +++-
 drivers/staging/hv/RingBuffer.c       |    3 ++-
 drivers/staging/hv/Sources.c          |   30 ------------------------------
 8 files changed, 17 insertions(+), 34 deletions(-)

--- a/drivers/staging/hv/Channel.c
+++ b/drivers/staging/hv/Channel.c
@@ -22,6 +22,7 @@
  */
 
 #include <linux/kernel.h>
+#include <linux/mm.h>
 #include "include/osd.h"
 #include "include/logging.h"
 
--- a/drivers/staging/hv/ChannelInterface.c
+++ b/drivers/staging/hv/ChannelInterface.c
@@ -21,6 +21,10 @@
  *
  */
 
+#include <linux/kernel.h>
+#include <linux/mm.h>
+#include "include/osd.h"
+
 #include "VmbusPrivate.h"
 
 static int
--- a/drivers/staging/hv/ChannelMgmt.c
+++ b/drivers/staging/hv/ChannelMgmt.c
@@ -22,6 +22,8 @@
  */
 
 
+#include <linux/kernel.h>
+#include <linux/mm.h>
 #include "include/osd.h"
 #include "include/logging.h"
 
--- a/drivers/staging/hv/Connection.c
+++ b/drivers/staging/hv/Connection.c
@@ -22,8 +22,10 @@
  */
 
 
+#include <linux/kernel.h>
+#include <linux/mm.h>
+#include <linux/vmalloc.h>
 #include "include/logging.h"
-
 #include "VmbusPrivate.h"
 
 /* Globals */
--- a/drivers/staging/hv/Hv.c
+++ b/drivers/staging/hv/Hv.c
@@ -21,8 +21,9 @@
  *
  */
 
+#include <linux/kernel.h>
+#include <linux/mm.h>
 #include <linux/vmalloc.h>
-#include <asm/io.h>
 #include "include/logging.h"
 #include "VmbusPrivate.h"
 
--- a/drivers/staging/hv/Makefile
+++ b/drivers/staging/hv/Makefile
@@ -3,7 +3,9 @@ obj-$(CONFIG_HYPERV_STORAGE)	+= hv_storv
 obj-$(CONFIG_HYPERV_BLOCK)	+= hv_blkvsc.o
 obj-$(CONFIG_HYPERV_NET)	+= hv_netvsc.o
 
-hv_vmbus-objs := vmbus_drv.o osd.o Sources.o
+hv_vmbus-objs := vmbus_drv.o osd.o \
+		 Vmbus.o Hv.o Connection.o Channel.o \
+		 ChannelMgmt.o ChannelInterface.o RingBuffer.o
 hv_storvsc-objs := storvsc_drv.o StorVsc.o
 hv_blkvsc-objs := blkvsc_drv.o BlkVsc.o
 hv_netvsc-objs := netvsc_drv.o NetVsc.o RndisFilter.o
--- a/drivers/staging/hv/RingBuffer.c
+++ b/drivers/staging/hv/RingBuffer.c
@@ -21,7 +21,8 @@
  *
  */
 
-
+#include <linux/kernel.h>
+#include <linux/mm.h>
 #include "include/logging.h"
 #include "RingBuffer.h"
 
--- a/drivers/staging/hv/Sources.c
+++ /dev/null
@@ -1,30 +0,0 @@
-/*
- *
- * Copyright (c) 2009, Microsoft Corporation.
- *
- * This program is free software; you can redistribute it and/or modify it
- * under the terms and conditions of the GNU General Public License,
- * version 2, as published by the Free Software Foundation.
- *
- * This program is distributed in the hope it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
- * more details.
- *
- * You should have received a copy of the GNU General Public License along with
- * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
- * Place - Suite 330, Boston, MA 02111-1307 USA.
- *
- * Authors:
- *   Haiyang Zhang <haiyangz@microsoft.com>
- *   Hank Janssen  <hjanssen@microsoft.com>
- *
- */
-
-#include "Vmbus.c"
-#include "Hv.c"
-#include "Connection.c"
-#include "Channel.c"
-#include "ChannelMgmt.c"
-#include "ChannelInterface.c"
-#include "RingBuffer.c"
